### PR TITLE
[IMP] sales: update customer addresses

### PR DESCRIPTION
**Pull request by meng-odoo**
Updated this doc for Odoo 16, and rewrote it for clarity and brevity. Replaced images, and fixed image names and alt descriptions.
Closes task #[3152071](https://www.odoo.com/web#id=3152071&cids=3&model=project.task&view_type=form) and Issue #[3693](https://github.com/odoo/documentation/issues/3693)

### CODE REVIEW

**Comment by meng-odoo** [comment]
@StraubCreative this is ready for your review

**Comment by StraubCreative** [comment]
@ksc-odoo can you plz take a look at this doc? Any improvements y'all can tackle before it gets to me will improve likelihood of quick merge. TIA!

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 1]
```diff
@@ -2,74 +2,54 @@
...
+With Odoo Sales, customers can use different addresses for delivery and invoicing.
 
 Activate the feature
 ====================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Customer Addresses**
-feature.
+Activate the :guilabel:`Customer Addresses` feature in the settings. It is located in
+:menuselection:`Accounting > Configuration > Settings` if the Accounting app is installed, or
+:menuselection:`Invoicing > Configuration > Settings` if the Accounting app is not installed.
```
Comment (ksc-odoo) [inline content/applications/sales/sales/send_quotations/different_addresses.rst:12]: Activate the :guilabel:`Customer Addresses` feature in the settings. 

If the :guilabel:`Accounting` app is installed, the :guilabel:`Customer Addresses` feature is located in :menuselection:`Accounting > Configuration > Settings`.

If the :guilabel:`Accounting` app is *not* installed, the :guilabel:`Customer Addresses` feature is located in
:menuselection:`Invoicing > Configuration > Settings`.

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 2]
```diff
@@ -2,74 +2,54 @@
 Deliver and invoice to different addresses
 ==========================================
 
-With Odoo Sales, you can configure different addresses for delivery and invoicing. For some
-customers, it will be very practical to define specific billing and shipping addresses. Indeed, not
-everyone will have the same delivery location as the invoicing location.
+With Odoo Sales, customers can use different addresses for delivery and invoicing.
```
Comment (ksc-odoo) [inline content/applications/sales/sales/send_quotations/different_addresses.rst:5]: With :guilabel:`Odoo Sales`, ...

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 3]
```diff
@@ -2,74 +2,54 @@
...
+==========================
 
-When you create a quotation, you must add a customer. This customer can be a company or a person
-with specific billing and shipping addresses already defined and registered in the system, or it can
-be a new customer. In this case, you have to **Create and edit** the contact form for your new
-customer and link it, if necessary, to a company. In this contact form, you will be able to add,
-delete and modify invoice and delivery addresses.
+Navigate to the :guilabel:`Contacts` application, or to :menuselection:`Sales > Orders >
+Customers`, and click on a customer to open their contact form. Under the :guilabel:`Contacts &
+Addresses` tab, click :guilabel:`Add`.
```
Comment (ksc-odoo) [inline content/applications/sales/sales/send_quotations/different_addresses.rst:24]: Navigate to the :guilabel:`Contacts` application (or to :menuselection:`Sales > Orders >
Customers`), and click on a customer to open their contact form. 

Under the :guilabel:`Contacts & Addresses` tab, click :guilabel:`Add`.

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 4]
```diff
@@ -2,74 +2,54 @@
...
-Deal with different addresses
-=============================
+   :alt: Create a new contact/address on a contact form
 
-Like for the previous example, go to :menuselection:`Sales --> Orders --> Customers` and create a
-new customer. There, you can add company information but, more importantly, you can enter billing
-and shipping addresses under the **Contacts & Addresses** tab.
+Enter the address information. Then, click :guilabel:`Save & Close` to save the address and close
+the :guilabel:`Create Contact` window, or click :guilabel:`Save & New` to save this address and
+immediately input another one.
```
Comment (ksc-odoo) [inline content/applications/sales/sales/send_quotations/different_addresses.rst:41]: Enter the address information. Then, click :guilabel:`Save & Close` to save the address, and close
the :guilabel:`Create Contact` window.

Or, click :guilabel:`Save & New` to save this address, and immediately input another one.

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 5]
```diff
@@ -2,74 +2,54 @@
...
-   :class: img-thumbnail
-   :alt: How to manage different customer addresses on Odoo Sales?
+Add addresses to the quotation
+==============================
 
-Once done, you can return to your Sales dashboard and create a new quotation. Now, if you enter
-your new customer, you will see that the other fields will fill in by themself with the information
-previously saved for billing and shipping addresses.
+When a customer is added to a quotation, the :guilabel:`Invoice Address` and :guilabel:`Delivery
+Address` will autopopulate according to the customer’s contact form.
```
Comment (ksc-odoo) [inline content/applications/sales/sales/send_quotations/different_addresses.rst:47]: When a customer is added to a quotation, the :guilabel:`Invoice Address` and :guilabel:`Delivery
Address` will autopopulate (according to the customer’s contact form).

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 6]
```diff
@@ -2,74 +2,54 @@
...
+.. image:: different_addresses/quotation-address-autopopulate.png
    :align: center
    :class: img-thumbnail
-   :alt: Automatic quotation fields filling on Odoo Sales
+   :alt: Invoice and Delivery Addresses autopopulate on a quotation
 
-With Odoo Sales, it is now very convenient to play with various addresses in terms of invoice and
-delivery features.
+The :guilabel:`Invoice Address` and :guilabel:`Delivery Address` can also be edited directly from
+the quotation by clicking on the :guilabel:`Internal link` button next to the address line.
```
Comment (ksc-odoo) [inline content/applications/sales/sales/send_quotations/different_addresses.rst:55]: The :guilabel:`Invoice Address` and :guilabel:`Delivery Address` can also be edited directly from
the quotation, by clicking on the :guilabel:`Internal link` button next to the address line.

**Review by ksc-odoo — COMMENTED** [review]
hey @meng-odoo - I just did a quick review of your updates to this doc (per ZST's instructions). Just a few minor modifications to consider before passing it along to @StraubCreative for final review. Great job!!!

**Comment by ksc-odoo** [comment]
> @ksc-odoo can you plz take a look at this doc? Any improvements y'all can tackle before it gets to me will improve likelihood of quick merge. TIA!

@StraubCreative done deal. just a few small things to adjust and then it should be good to go. Let me know if there's anything else you need from me. Thanks!

**Comment by meng-odoo** [comment]
Thanks for the review @ksc-odoo! I've incorporated your suggestions on the latest commit.
@StraubCreative this is ready for you.

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 7]
```diff
@@ -2,74 +2,59 @@
...
+immediately input another one.
+
+Add addresses to the quotation
+==============================
 
-Once done, you can return to your Sales dashboard and create a new quotation. Now, if you enter
-your new customer, you will see that the other fields will fill in by themself with the information
-previously saved for billing and shipping addresses.
+When a customer is added to a quotation, the :guilabel:`Invoice Address` and :guilabel:`Delivery
+Address` will autopopulate according to the customer’s contact form.
```
Comment (StraubCreative) [inline content/applications/sales/sales/send_quotations/different_addresses.rst:52]: Slightly more specific
```suggestion
Address` fields will autopopulate according to the values that were added on the customer's contact
form.
```

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 8]
```diff
@@ -2,74 +2,59 @@
...
+.. image:: different_addresses/quotation-address-autopopulate.png
    :align: center
    :class: img-thumbnail
-   :alt: Automatic quotation fields filling on Odoo Sales
+   :alt: Invoice and Delivery Addresses autopopulate on a quotation
 
-With Odoo Sales, it is now very convenient to play with various addresses in terms of invoice and
-delivery features.
+The :guilabel:`Invoice Address` and :guilabel:`Delivery Address` can also be edited directly from
+the quotation, by clicking on the :guilabel:`Internal link` button next to the address line.
```
Comment (StraubCreative) [inline content/applications/sales/sales/send_quotations/different_addresses.rst:60]: Seems like the doc ends a little prematurely. It's not quite clear what the outcome (dare I say, "wow" moment) is. Now that these addresses are configured, what now? What outcome is produced?

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @meng-odoo 
Couple change requests here.
I like that the doc is simple and provides clear + linear instruction, however I think we can be more specific in some spots. See comments below.

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 9]
```diff
@@ -2,74 +2,59 @@
 Deliver and invoice to different addresses
 ==========================================
 
-With Odoo Sales, you can configure different addresses for delivery and invoicing. For some
-customers, it will be very practical to define specific billing and shipping addresses. Indeed, not
-everyone will have the same delivery location as the invoicing location.
+With :guilabel:`Odoo Sales`, customers can use different addresses for delivery and invoicing.
```
[the remark this answers is not in the document]
Comment (meng-odoo) [reply content/applications/sales/sales/send_quotations/different_addresses.rst:5]: By "customers" I actually meant customers in the Contacts app, not the Odoo user. Clarified in the latest version.

**Comment by meng-odoo** [comment]
@StraubCreative whoops we never finalized this. I incorporated your changes in the latest commit. If you want to go ahead and merge, we can add this to Q1 deliverables :)

**Review by StraubCreative — APPROVED** [review]
LGTM. Short and sweet 🙂 

@robodoo r+

**Pull request closed without merging**